test/odr_test.py

```diff
@@ -0,0 +1,48 @@
+import os
+import sys
+import re
+
+this_path = os.path.dirname(__file__)
+
+all_headers = set()
+include_path = this_path + "/../include/"
+for root, dirs, files in os.walk(include_path):
+    for fn in files:
+        fn = root + "/" + fn
+        assert fn.startswith(include_path)
```

Comment:
I would probably make sure these are resolved with `os.path.abspath` first. Might be fine as is though.

Reply:
What difference would abspath make?